portfolio/src/main/webapp/index.html

```diff
@@ -58,11 +57,53 @@ <h3 id="hobbies"> Hobbies </h3>
 
       <hr>
 
-      <h3 id="contact"> Contact Me </h3>
-      <a target="_blank" href="https://github.com/Kaleab470">Github</a>
-      <a target="_blank" href="https://linkedin.com/in/kaleab-gebremichael">LinkedIn</a>
-      <a target="_blank" href="mailto:kaleabsolomon470@gmail.com">Email</a>
+      <h3 id="more"> More about me </h3>
+
+      <p>My favorite actor you ask?</p>
+      <img class="photo" src="Images/Jim-1.jpg">
+      <br>
```

Comment:
Are hr tags off limits as well?

Reply:
I think hr works better

For context the reasoning behind this is the preference for "Semantic HTML"
https://www.w3schools.com/html/html5_semantic_elements.asp

Ideally, the page should make sense just by reading it. To accomplish that, you want the element types you choose to convey meaning.  br is usually avoided because the only meaning is that there is a line break, and a line break is very rarely actually important to the meaning.  Youre using it here to separate sections, which I think an hr tag conveys a lot better than br, which could mean anything from "these should be on a different line" to "there should be a gap here" to tons of other gross uses of br that are out there.